### PR TITLE
Fixes 219 Provide a consistent default port

### LINKED ISSUE #219: Inconsistency in usage when it comes to port 3000
The latest release v1.62.0 defaults to port `8000` when used with `./server.sh` and documentation still takes about port `3000`. While the Docker image, correctly stated in the documents, uses port `3000`.

It might be a suggestion to add `--port 3000` to the server.sh file for the release, just like is done for the Docker image.

### PR DESCRIPTION
**Pull request by gbraad**
This PR fixes #219 

It provides a default value for the port consistent with the documentation

### CODE REVIEW

**Comment by gbraad** [comment]
This fails on: https://github.com/microsoft/vscode/blob/822f995357f725216cdee6e05f1f1552ccbbd882/src/vs/server/remoteExtensionHostAgent.ts#L21-L29


`$ server.sh --port 8000`
```
Extension host agent listening on 8000

Option port can only be defined once. Using value 3000.
```
... but is actually listening on `8000` as requested.

**Comment by gbraad** [comment]
this will however show an error with the Dockerfile because multiple ports are specified. Best to update the documentation instead or remove this from the Dockerfile.

File: scripts/sync-with-upstream.sh [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+upstream_url="https://github.com/microsoft/vscode.git"
+upstream_branch=${1:-"upstream/main"}
+local_branch=${2:-"main"}
+base_commit_msg=${3:-"code web server initial commit"}
+only_sync=${4:-"true"}
+
+exit_script() {
+	reason=$1
+	echo "Update script ended unsucessfully"
+	echo "Reason: $reason"
```
Comment (iamtangram) [inline scripts/sync-with-upstream.sh:14]: ccc

**Review by iamtangram — COMMENTED** [review]
csdcsdc

**Review by iamtangram — CHANGES_REQUESTED** [review]
csdcsdc

File: scripts/sync-with-upstream.sh [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	exit 1
+}
+
+if [[ "$OSTYPE" == "darwin"* ]]; then
+	realpath() { [[ $1 = /* ]] && echo "$1" || echo "$PWD/${1#./}"; }
+	ROOT=$(dirname $(dirname $(realpath "$0")))
+else
+	ROOT=$(dirname $(dirname $(readlink -f $0)))
+	# --disable-dev-shm-usage --use-gl=swiftshader: when run on docker containers where size of /dev/shm
+	# partition < 64MB which causes OOM failure for chromium compositor that uses the partition for shared memory
```
Comment (iamtangram) [inline scripts/sync-with-upstream.sh:24]: cdscds

Comment (iamtangram) [reply scripts/sync-with-upstream.sh:24]: vvvvvv

**Review by jeanp413 — APPROVED** [review]
Thanks!

**Pull request merged**